{"body": "<p>Finally i found source of my stupid error. Reason was that earlier i had photo field in my model. But after making changes in model i forget to make migrations.</p>\n", "tags": ["python", "django", "django-rest-framework"], "creation_date": 1476378185, "score": 0, "last_activity_date": 1476378185, "answer_id": 40026977, "is_accepted": true, "owner": {"user_id": 6940208, "reputation": 20, "user_type": "registered", "accept_rate": 80, "display_name": "Robin", "link": "http://stackoverflow.com/users/6940208/robin", "profile_image": "https://www.gravatar.com/avatar/931804c575c3550365d660d4c7cfdac8?s=128&d=identicon&r=PG&f=1"}, "title": "The field was declared on serializer , but has not been included error in django rest", "question_id": 39928879}